exceptions/exceptions.hh

```diff
@@ -203,6 +204,12 @@ struct read_failure_exception : public request_failure_exception {
     { }
 };
 
+class overflow_error_exception: public cassandra_exception {
+public:
+    overflow_error_exception(sstring msg) noexcept
+        : cassandra_exception(exception_code::UNAUTHORIZED, std::move(msg)) {}
```

Comment:
UNAUTHORIZED?

Reply:
sorry, not sure how this came to be this way.